### PR TITLE
Arcade Validation should only look at product repo latest builds instead of LKGs

### PR DESCRIPTION
**Pull request by missymessa**
We'll use the Latest Build endpoint from AzDO's API to determine if we should validate Arcade against the repo. If the Latest Build result is "partiallySucceeded" or "succeeded", then we'll use the SHA of that build to build Arcade with in that repo. If the result is "cancelled" or "failed", then we won't validate Arcade with that repo.

### CODE REVIEW

**Comment by missymessa** [comment]
Validated this code here: https://dnceng.visualstudio.com/internal/_build/results?buildId=635526&view=results

(A couple notes: Removed the DEBUG write host statement for the URI, and changed the "bail out" message to Write-Error instead of Write-Warning so that it will fail the job properly.)

File: eng/validation/build-arcadewithrepo.ps1 [hunk 1]
```diff
@@ -45,64 +44,32 @@ $testRootBase = if ($env:AGENT_WORKFOLDER) { $env:AGENT_WORKFOLDER } else { $([S
...
-    ## Have there been any builds in the last $daysOfOldestBuild days? 
+    ## Verified that this API gets completed builds, not in progress builds
     $headers = Get-AzDOHeaders
-    $count = 0
+    $uri = "https://dev.azure.com/${global:azdoOrg}/${global:azdoProject}/_apis/build/latest/${global:buildDefinitionId}?branchName=${global:subscribedBranchName}&api-version=5.1-preview.1"
+    $response = (Invoke-WebRequest -Uri $uri -Headers $headers -Method Get) | ConvertFrom-Json
 
-    foreach($reason in $global:buildReasonsList)
+    ## Is it successful or partially successful? Then use that as the foundation for our branch. 
+    ## How do we want to handle cancelled builds?
```
Comment (riarenas) [inline eng/validation/build-arcadewithrepo.ps1:55]: I think we already have the answer to this question in the code below: we handle them as not successful

Comment (missymessa) [reply eng/validation/build-arcadewithrepo.ps1:55]: I'll take that note to self out of there 😂

**Comment by missymessa** [comment]
Looks like Arcade Validation is looking at the latest completed build instead of looking for a LKG in the last three days. Closing.

**Pull request merged**